ui/app/pages/send/send.utils.js

```diff
@@ -236,11 +236,6 @@ async function estimateGas ({
     }
   }
 
-  // if not, fall back to block gasLimit
-  if (!blockGasLimit) {
```

Comment:
I realize this is redundant with the default parameter, but deleting this is still a functional change for the case where `null` or empty string is passed in rather than `undefined`. And it turns out the default value for `blockGasLimit` is an empty string (see `currentBlockGasLimit` in the account tracker).

Reply:
Good point, thanks